flutter_restaruant/flutter_restaruant/Makefile

```
@@ -1,70 +1,118 @@
 
-run_profile_all:
-	@flutter run --profile -d all
-
-run_release_all:
-	@flutter run --release -d all
+pod:
+	@cd ios && pod install
 
-## Clean the environment.
-clean: 
+clean:
```

checkmake (0.3.2)

[warning] Target body for "clean" exceeds allowed length of 5 lines (10).

(maxbodylength)